src/constants/global-constants.ts

```diff
@@ -0,0 +1 @@
+export const IS_DOCUMENT = typeof document !== 'undefined';
```

Comment:
이 상수를 만드신 목적이 실행 환경이 서버인지 클라이언트인지 확인하시는 것이라면
이렇게 쓰는 것으로 더 많은 곳에서 사용할 수 있을 것 같습니다!

```suggestion
export const isServer = typeof window !== 'undefined';
```

Reply:
아하 넵 수정하겠습니다!